### PR TITLE
reQueueMovie and getMovieStatus endpoints

### LINKED ISSUE #10: Movies - reQueueMovie endpoint
Endpoint details: https://api.helioviewer.org/docs/v2/api/api_groups/movies.html#id2

### LINKED ISSUE #11: Movies - getMovieStatus endpoint
Endpoint Details: https://api.helioviewer.org/docs/v2/api/api_groups/movies.html#id8

### PR DESCRIPTION
**Pull request by akash5100**
* closes #10 
* closes #11

### CODE REVIEW

File: hvpy/api_groups/movies/re_queue_movie.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+class reQueueMovieInputParameters(HvpyParameters):
+    """
+    Handles the input parameters of the ``reQueueMovie`` API.
```
Comment (nabobalis) [inline hvpy/api_groups/movies/re_queue_movie.py:8]: This is not a, fix it in this PR.

For a future issue/PR we should actually turn the reference into a named url and then link it up here.

File: hvpy/api_groups/movies/get_movie_status.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    {Shared}
+    id
+        Unique movie identifier, returned as a response by the ``queueMovie`` endpoint request.
+    format
+        Movie format.
+    verbose
+        Optionally include extra metadata in the response.
+        Defaults to ``False``.
+    calback
+        Optionally wrap the response in a callback function.
```
Comment (nabobalis) [inline hvpy/api_groups/movies/get_movie_status.py:21]: This should follow the other callback examples for documentation.

File: hvpy/api_groups/movies/get_movie_status.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    token
+        API token.
```
Comment (nabobalis) [inline hvpy/api_groups/movies/get_movie_status.py:23]: Missing the default.

File: hvpy/api_groups/movies/get_movie_status.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    Attributes
+    ----------
+    {Shared}
+    id
+        Unique movie identifier, returned as a response by the ``queueMovie`` endpoint request.
+    format
+        Movie format.
+    verbose
+        Optionally include extra metadata in the response.
+        Defaults to ``False``.
```
Comment (nabobalis) [inline hvpy/api_groups/movies/get_movie_status.py:19]:
```suggestion
        Defaults to `False`.
```

File: hvpy/facade.py [hunk 5]
```diff
@@ -382,3 +384,57 @@ def queueMovie(
...
+    Re-generate a custom movie that is no longer cached on the server.
+
+    Parameters
+    ----------
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import reQueueMovie
+    >>> reQueueMovie(id="gxRN5", force=True)
+    {'id': 'gxRN5', 'eta': 274, 'queue': 0, 'token': '...'}
```
Comment (nabobalis) [inline hvpy/facade.py:404]: Is this ID and ETA fixed?

File: hvpy/facade.py [hunk 6]
```diff
@@ -382,3 +384,57 @@ def queueMovie(
...
+    """
+    Get the status of a movie.
+
+    Parameters
+    ----------
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import getMovieStatus
+    >>> getMovieStatus(id="VXvX5", format="mp4")
```
Comment (nabobalis) [inline hvpy/facade.py:430]: Will this ID always be there?

Comment (wafels) [reply hvpy/facade.py:430]: Movies that are more than 90 days old are deleted, but they can be re-built. The ID should always be there as a reference to movie.

**Review by nabobalis — APPROVED** [review]

**Review by dgarciabriseno — APPROVED** [review]

**Pull request merged**